DependencyInjection/HttplugExtension.php

```diff
@@ -54,6 +66,10 @@ protected function configureClients(ContainerBuilder $container, array $config)
                 $first = $name;
             }
 
+            if (isset($config['_inject_collector_plugin'])) {
+                array_unshift($arguments['plugins'], 'httplug.collector.history_plugin');
```

Comment:
actually, adds the plugin first. does that mean we log before anything else? that will mean that if we have the redirect plugin active, we will not see that a redirect happened. then again, if redirect was activated, we might not care about it.
but it also means we will not log the request exactly as it was sent, with base url or credentials or whatever was added by the plugins. do we maybe need a flag to decide whether to add to the beginning or to the end?